build/dependencies.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
-    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <CoreFxVersion>4.4.0-*</CoreFxVersion>
```

Comment:
Yes, this should be bumped. The 4.4.0 packages often include new ns2.0 runtime libraries or reference assemblies